data_steward/validation/participants/identity_match.py

```diff
@@ -669,15 +677,22 @@ def write_results_to_drc_bucket(project, validation_dataset=None):
 
     date_string = _get_date_string(validation_dataset)
     hpo_sites = readers.get_hpo_site_names()
-    # generate aggregate site report
-    bucket = gcs_utils.get_drc_bucket()
-    filename = os.path.join(validation_dataset,
-                            consts.REPORT_DIRECTORY.format(date=date_string),
-                            consts.REPORT_TITLE)
-    _, errors = writers.create_site_validation_report(project,
-                                                      validation_dataset,
-                                                      hpo_sites, bucket,
-                                                      filename)
+    filename: str = os.path.join(
+        validation_dataset, consts.REPORT_DIRECTORY.format(date=date_string),
+        consts.REPORT_TITLE)
+    try:
```

Comment:
Removal of strings in place of buckets was added here.  Exception handling added that previously didn't exitst.